### ISSUE TITLE
Provide the actual bad string being used

### ISSUE THREAD

**Issue by lirantal** [lirantal/lockfile-lint#17]
**Is your feature request related to a problem? Please describe.**
Right now when the linter fails on an untrusted option found in the lockfile it says which package is it, but it doesn't say what is the actual bad string being used.

For example, an output of lockfile-lint could be:

```bash
detected invalid scheme for package: @babel/code-frame@^7.0.0
error: command failed with exit code 1
```

But it doesn't say what is the string provided as the "scheme" that is found to be invalid, and so it would be a nice DX to also print this information out with regards to the current validators we have.

**Describe the solution you'd like**
This is probably something we want to fix in the way that the `lockfile-lint-api` works and provide this information, at which point the CLI package `lockfile-lint` can then use it.

**Comment by pratik97** [comment]
I will take a look :)

**Comment by pratik97** [comment]
@lirantal raised a PR for this

**Comment by lirantal** [comment]
thanks!
